src/NuGet.Core/NuGet.Packaging/PackagePathResolver.cs

```diff
@@ -25,6 +25,13 @@ public PackagePathResolver(string rootDirectory, bool useSideBySidePaths = true)
                     string.Format(Strings.StringCannotBeNullOrEmpty, nameof(rootDirectory)),
                     nameof(rootDirectory));
             }
+            if (!Path.IsPathRooted(rootDirectory))
+            {
+                throw new ArgumentException(
+                    string.Format(Strings.MustContainAbsolutePath, nameof(rootDirectory), string.Empty),
```

Comment:
The 2nd parameter in the format should not be null, but the actual path so that the error is actionable.